nn4mc/parser/onnx_parser/_onnxparser.py

```diff
@@ -0,0 +1,27 @@
+from nn4mc.parser._parser import Parser
```

Comment:
TODO(sarahaguasvivas): Use `test_translator` to see if nn4mc doesn't crash with these models:

https://github.com/correlllab/nn4mc_py/blob/master/tests/test_translator/test_full.py

Create a new one called `test_full_onnx.py`